torch/_torch_docs.py

```diff
@@ -2311,10 +2311,16 @@ def merge_dicts(*dicts):
 .. math::
     inv = (u^T u)^{{-1}}
 
+Supports input of float, double, cfloat and cdouble dtypes. Also supports batches
+of vectors, for which it computes the product along the dimension :attr:`dim`.
```

Comment:
This should be batch of matrices, not vectors. Also there's no `dim` arg for this function. Refer to [cholesky docs](https://pytorch.org/docs/master/generated/torch.linalg.cholesky.html?highlight=cholesky#torch.linalg.cholesky) as an example.

Reply:
You're right. I reflected all now.